docs/cmd/tkn_pipelinerun.md

```diff
@@ -0,0 +1,41 @@
+## tkn pipelinerun
+
+Pipelinerun command group
+
+### Synopsis
+
+Pipelinerun command group
+
+### Aliases
+
+```
+pipelinerun, pr, pipelineruns
+```
+
+### Available Commands
+
+```
+describe    Describe a pipelinerun in a namespace
```

Comment:
Is it PipelineRun, pipelinerun, or Pipelinerun

It should be consistent when referring to it.

Reply:
the command is  pipelinerun